lib/build.py

```diff
@@ -976,6 +999,53 @@ def __init__(self, *args):
       self.value = self.tree.terminals_cat("LINE_CHUNK")
 
 
+class Label(Instruction):
+
+   __slots__ = ("key",
+                "value")
+
+   def __init__(self, *args):
+      super().__init__(*args)
+      self.commit_files |= {ch.Path("ch/metadata.json")}
+
+   @property
+   def str_(self):
+      return "%s='%s'" % (self.key, self.value)
+
+   def execute(self):
+      with (self.image.unpack_path // "/ch/metadata.json").open_("wt") \
+           as fp:
+         for (k, v) in self.label_label.items():
+            print("%s=%s" % (k, v), file=fp)
```

Comment:
This class doesn't need `execute()`; see point 3 in the comment for `prepare()` above.

Reply:
Thanks, another good catch